[{"name": "test.test_sample_from_concatenation.TestSampleFromConcatenation.test_deterministic", "status": "failed", "message": "src.sample_from_concatenation.MarkovChain does not exist!", "passed": false, "points": ["p07-17.1"], "backtrace": ["  File \"/anaconda3/lib/python3.6/unittest/case.py\", line 59, in testPartExecutor\n    yield\n", "  File \"/anaconda3/lib/python3.6/unittest/case.py\", line 605, in run\n    testMethod()\n", "  File \"/Users/satu/Kurssit/hy-data-analysis-with-python-spring-2019/part07-e17_sample_from_concatenation/test/test_sample_from_concatenation.py\", line 42, in test_deterministic\n    mc = MarkovChain(s, k)\n", "  File \"/Users/satu/Kurssit/hy-data-analysis-with-python-spring-2019/part07-e17_sample_from_concatenation/tmc/utils.py\", line 10, in fail\n    raise AssertionError(err)\n"]}, {"name": "test.test_sample_from_concatenation.TestSampleFromConcatenation.test_length_one", "status": "failed", "message": "src.sample_from_concatenation.MarkovChain does not exist!", "passed": false, "points": ["p07-17.1"], "backtrace": ["  File \"/anaconda3/lib/python3.6/unittest/case.py\", line 59, in testPartExecutor\n    yield\n", "  File \"/anaconda3/lib/python3.6/unittest/case.py\", line 605, in run\n    testMethod()\n", "  File \"/Users/satu/Kurssit/hy-data-analysis-with-python-spring-2019/part07-e17_sample_from_concatenation/test/test_sample_from_concatenation.py\", line 34, in test_length_one\n    mc = MarkovChain(s, k)\n", "  File \"/Users/satu/Kurssit/hy-data-analysis-with-python-spring-2019/part07-e17_sample_from_concatenation/tmc/utils.py\", line 10, in fail\n    raise AssertionError(err)\n"]}, {"name": "test.test_sample_from_concatenation.TestSampleFromConcatenation.test_second", "status": "failed", "message": "src.sample_from_concatenation.MarkovChain does not exist!", "passed": false, "points": ["p07-17.1"], "backtrace": ["  File \"/anaconda3/lib/python3.6/unittest/case.py\", line 59, in testPartExecutor\n    yield\n", "  File \"/anaconda3/lib/python3.6/unittest/case.py\", line 605, in run\n    testMethod()\n", "  File \"/Users/satu/Kurssit/hy-data-analysis-with-python-spring-2019/part07-e17_sample_from_concatenation/test/test_sample_from_concatenation.py\", line 52, in test_second\n    mc = MarkovChain(s, k)\n", "  File \"/Users/satu/Kurssit/hy-data-analysis-with-python-spring-2019/part07-e17_sample_from_concatenation/tmc/utils.py\", line 10, in fail\n    raise AssertionError(err)\n"]}]